ccm_web/client/src/utils/enrollmentValidators.ts

```diff
@@ -0,0 +1,113 @@
+import { ClientEnrollmentType, isValidRole } from '../models/canvas'
```

Comment:
Some of the repetitive code in this file could be generalized see my comment https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/270/files#r780433576

Reply:
How's this? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/270/commits/7238bc99ebf59b82dfaa6fefb40aeb045d37368b

Reply:
Nice!